synfig-studio/src/gui/app.cpp

```diff
@@ -2371,8 +2371,16 @@ App::dialog_open_file_ext(const std::string &title, std::vector<std::string> &fi
 
 	dialog->set_transient_for(*App::main_window);
 	dialog->set_current_folder(prev_path);
-	dialog->add_button(_("Cancel"), Gtk::RESPONSE_CANCEL)->set_image_from_icon_name("gtk-cancel", Gtk::ICON_SIZE_BUTTON);
-	dialog->add_button(_("Import"), Gtk::RESPONSE_ACCEPT)->set_image_from_icon_name("gtk-open",   Gtk::ICON_SIZE_BUTTON);
+
+	//Add action button colors
```

Comment:
As you may have noticed, you are doing a lot of repetitive actions here. To make the code cleaner, I suggest adding 2 new functions: 

1. `void button_style_cancel(Gtk::Button *button, const Glib::ustring& icon_name);`
2. `void button_style_suggest(Gtk::Button *button, const Glib::ustring& icon_name);`

In this case, you can make the code much cleaner:
Before:

```
	Gtk::Button *button = dialog->add_button(_("Cancel"), Gtk::RESPONSE_CANCEL);//->set_image_from_icon_name("gtk-cancel", Gtk::ICON_SIZE_BUTTON);
	button->set_image_from_icon_name("gtk-cancel", Gtk::ICON_SIZE_BUTTON);
	button->get_style_context()->add_class("destructive-action");

	button = dialog->add_button(_("Import"), Gtk::RESPONSE_ACCEPT);//->set_image_from_icon_name("gtk-open",   Gtk::ICON_SIZE_BUTTON);
	button->set_image_from_icon_name("gtk-open",   Gtk::ICON_SIZE_BUTTON);
	button->get_style_context()->add_class("suggested-action");
```

After:
```
	button_style_cancel(dialog->add_button(_("Cancel"), Gtk::RESPONSE_CANCEL), "gtk-cancel");
	button_style_suggest(dialog->add_button(_("Import"), Gtk::RESPONSE_ACCEPT), "gtk-open");
```

This will also allow you to change the style from one place.

Reply:
@ice0 thank you for the suggestion. I had thought about this, and glad to hear it from you too.

Reply:
@ice0 I reduced line of code with the suggestions you made.  But I also opened another pull request ref #2498 because Rodolfo suggested we add "destructive-action" buttons only when something could be destructive like exiting without saving.  Can you check out the PR and see if that is a better idea than coloring all the buttons in ```FileChooserDialog``` boxes.  Thank you for you time.

Reply:
@nickleus27 
I agree, that makes sense.